source/clock.rst

```diff
@@ -0,0 +1,143 @@
+==========
+Clock
+==========
+
+About
+-----
+
+The Clock activity is an education and teaching tool to help children learn how to read time. This gives them a way to know the time with their XO even if they do not own a watch.
+
+.. image :: ../images/Clock-img1.png
+
+.. image :: ../images/Clock-img2.png
+
+.. image :: ../images/Clock-img3.png
+
+Using Clock
+----------------
+
+Basic Functionality
+::::::::::
+Click on the Clock toolbar buttons to change the display mode and options.
+
+Display modes and options that are available:
+
+1. Simple analog display with hours numbers
```

Comment:
"Simple" here is redundant.  Also, this mode is an analog clock face.  Perhaps a reference to Wikipedia for definition of a clock face?